src/TLuaInterpreter.cpp

```diff
@@ -4911,7 +4913,7 @@ int TLuaInterpreter::getRooms( lua_State *L )
 // returned.
 int TLuaInterpreter::getAreaExits( lua_State *L )
 {
-    int area;
+    int area = 0;
```

Comment:
Technically, not required as it can never be used without initialisation, the execution flow that bypasses assigning a value ends with a `lua_error(L)` and we both know what _that_ means! :grinning: